engine-tests/src/test/java/org/terasology/persistence/internal/StorageManagerTest.java

```diff
@@ -129,9 +127,9 @@ public void setup() throws Exception {
         testBlock2 = context.get(BlockManager.class).getBlock("test:testblock2");
 
         context.put(ChunkProvider.class, mock(ChunkProvider.class));
-        BiomeManager biomeManager = mock(BiomeManager.class);
-        when(biomeManager.getBiomes()).thenReturn(Collections.<Biome>emptyList());
-        context.put(BiomeManager.class, biomeManager);
+        BiomeManager biomeManagerLocal = mock(BiomeManager.class);
```

Comment:
This is a mock object used for testing purposes. Call it for what it is! "mockBiomeManager"!